packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -0,0 +1,410 @@
+<template>
+    <div
+        ref="cookieBanner"
+        :class="[$style['c-cookieBanner'], $style['c-cookieBanner-overlay'], { [$style['c-cookieBanner--hidden']]: hideBanner }]"
+        attr="data-cookie-consent-overlay"
```

Comment:
What is this `attr` exactly? Maybe a stupid question from my side.

Reply:
Can you add `data-test-id="cookieBanner-component"` here please?

Reply:
`attr=` removed and `data-test-id="cookieBanner-component"` added